### DISCUSSION TITLE
S21 usage from hotel control modules?

### DISCUSSION THREAD

**Discussion by RaW79** [revk/xESP32-Faikout#520] [General]
Hi
is there anything known which S21 functions are used by the available Hotel control modules from Daikin e.g. Madoka?

They are advertising a enery control function and a block mode in case of room occupation or by key card.

I´m interested in the functions which are used in the backround for e.g. better photovoltaic integration. Is there sth. like a general block mode (key card) I/we could use by Faikin integration to block the Daikin operation by software for example in the case of no PV production.

**Comment by Sonic-Amiga** [comment]
Hello! Never seen Madoka. But here i have a BRP069C41 controller, which supports X50 protocol; and it has an option to sync with wired control (no idea what exactly it syncs), and it presents a choice of two control panels, one being Madoka.

So, i guess, those A/Cs use X50 protocol; and you can do all the things, supported by X50.

I don't have such conditioner, the controller was donated for reverse engineering and i am running it with a simulator.

**Reply by revk** [reply]
My ducted units have the madoka wall control (which I obviously don't use) and talk X50.

**Reply by RaW79** [reply]
Thanks. So no additional options for Faikin/MQTT (S21) control for energy magement incl. total stop/interlock of the aircon by this?
